examples/notebooks/Differential_Dependencies.ipynb

```diff
@@ -0,0 +1,3002 @@
+            }
+          },
+          "metadata": {},
+          "execution_count": 15
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "The difference table defines the search space for DDs. That means, the algorithm searches only for DDs constructed\n",
+        "from distance constraints stated in the difference table. Therefore, as you can see from the discovered DDs, all\n",
+        "of the distance constraints that were used there are stated in the difference table."
+      ],
+      "metadata": {
+        "id": "6sCz2MSXHkpu"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "for dd in dds:\n",
+        "  print(dd)"
+      ],
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "O5w-egMaHsNy",
+        "outputId": "8938abbc-0892-470f-a1ef-e3d5574ccf10"
+      },
+      "execution_count": 16,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "Departure [0, 0] ; Arrival [0, 0] -> Distance [0, 50]\n",
+            "Distance [0, 50] -> Duration [0, 15]\n",
+            "Departure [0, 3] ; Arrival [0, 3] -> Duration [0, 15]\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "The number of constraints for each column can be different. The difference table can be accepted by the algorithm\n",
+        "only in the format stated above. Note that different difference tables processed by the algorithm result in\n",
```

Comment:
I think this one will be better:

Please note that different difference tables fed into the algorithm result in different sections of the search space being explored and, thus, yield different results.